### ISSUE TITLE
Accessibility Issue: Duplicate IDs on one page

### ISSUE THREAD

**Issue by wakingmorning** [badcamp/badcamp-2018#167]
**Issue:** There are duplicate IDs on one page of the site.

**URL:** [Proposed Sessions](https://2018.badcamp.net/sessions/proposed)

**Duplicate ID:** `edit-skill--wrapper`

**Proposed Solution:** ID's should be unique. Duplicate IDs can cause issues with screen readers.
